ddtrace/contrib/botocore/patch.py

```diff
@@ -165,8 +168,17 @@ def inject_trace_to_eventbridge_detail(params, span):
         entry["Detail"] = detail_json
 
 
+def get_json_from_str(data_str):
+    # type: (str) -> Tuple[str, Optional[Dict[str, Any]]]
+    data_obj = json.loads(data_str)
+
+    if data_str.endswith(LINE_BREAK):
+        return LINE_BREAK, data_obj
+    return None, data_obj
```

Comment:
typing nit. We skip type checking on modules in `ddtrace/contrib` so mypy missed this. 

```suggestion
    return "", data_obj
```